workshops/models.py

```diff
@@ -1215,7 +1215,6 @@ class EventRequest(AssignmentMixin, ActiveMixin, CreatedUpdatedMixin,
 
     ADMIN_FEE_PAYMENT_CHOICES = (
         ('NP1', 'Non-profit / non-partner: US$2500'),
-        ('partner', 'Partner: US$1250'),
```

Comment:
What happens when we have a historic data with "partner" option in this table? Shouldn't we "hide" it from the form only?